python/mxnet/_numpy_op_doc.py

```diff
@@ -173,3 +217,255 @@ def _np_cumsum(a, axis=None, dtype=None, out=None):
         `axis` is not None or `a` is a 1-d array.
     """
     pass
+
+
+def _np_reciprocal(x, out=None, **kwargs):
```

Comment:
Seems you have changed `_np_reciprocal` to `_npi_reciprocal` in C++ and wrapped it in frontend. Then no need to add this here.